packages/protocol/contracts/bridge/Bridge.sol

```diff
@@ -582,6 +571,39 @@ contract Bridge is EssentialContract, IBridge {
         }
     }
 
+    /// @notice Consumes a given amount of Ether from quota manager.
+    /// @param _amount The amount of Ether to consume.
+    /// @return true if quota manager has unlimited quota for Ether or the given amount of Ether is
+    /// consumed already.
+    function _consumeEtherQuota(uint256 _amount) private returns (bool) {
```

Comment:
`_consumeEtherQuota` and `_loadContext` were moved in the file but their implementations remain eactly the same. No need to review them.